CDCgov/prime-reportstream
# Add `.limit()` to queries to bound them

Fixes #6304

Test Steps:
1. Query for a large amount of data (e.g. `api/adm/listreceiversconnstatus?start_date=2020-01-01T00:00:00.000Z`)
2. Verify only 9000 rows are returned. (Note most localhost devs don't have enough much data in them.)
Using UI
1. Log in as a Prime Admin
2. Go to `admin/send-dash` (menu: Admin-> Receiver Status Dashboard)
3. Select an earlier starting date (say 5 weeks back).
WARNING: HTML Render of this much data is slow in the browser. (separate client-side fix pending).
4. Check query only returns 9000 rows.

## Changes
- Added a `.limit(9000)` to two queries: 

## Checklist

### Testing
[ ] ~Added tests?~ Not sure how to test a limit of 9000 rows?

### Process

## Linked Issues
- Fixes #6304

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #6288 and #6303

## Specific Security-related subjects a reviewer should pay specific attention to
> (No to all)
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
